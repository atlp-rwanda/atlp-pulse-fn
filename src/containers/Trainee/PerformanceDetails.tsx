/* eslint-disable */
import React from 'react';
import TraineePerformanceDetails from '../../components/TraineePerformanceDetails';
import TraineeComments from '../../components/TraineeComments';
import useDocumentTitle from '../../hook/useDocumentTitle';

function PerformanceDetails() {
  useDocumentTitle('Performance details');
  return (
    <div className="bg-neutral-100 dark:bg-dark-frame-bg">
      <TraineePerformanceDetails />
      <TraineeComments />
    </div>
  );
}

export default PerformanceDetails;
