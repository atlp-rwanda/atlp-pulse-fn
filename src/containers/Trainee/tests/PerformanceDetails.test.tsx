/* eslint-disable */
import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import PerformanceDetails from '../PerformanceDetails';

describe('Detailed Peformance page', () => {
  it('Renders a detailed performance Page ', () => {
    const elem = renderer
      .create(
        <MemoryRouter>
          <PerformanceDetails />
        </MemoryRouter>,
      )
      .toJSON();
    expect(elem).toMatchSnapshot();
  });
});
