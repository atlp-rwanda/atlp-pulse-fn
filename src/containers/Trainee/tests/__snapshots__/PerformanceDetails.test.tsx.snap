// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Detailed Peformance page Renders a detailed performance Page  1`] = `
<div>
  <div>
    <div
      className="bg-neutral-100  dark:bg-dark-frame-bg md:flex sm:hidden flex-col justify-start items-center  min-h-screen "
    >
      <table
        className="lg:w-9/12 md:w-11/12 lg:h-[70%] md:h-[60%] md:ml-0 lg:ml-32 dark:bg-dark-bg shadow-lg px-5 py-8 rounded-md mt-32 "
      >
        <thead>
          <tr
            className="text-gray-300  bg-gray-100  dark:bg-dark-tertiary"
          >
            <th
              className="lg:py-3 md:py-2 lg:px-3 md:px-2  text-left  text-[#6B7280] dark:text-dark-text-fill "
            >
              Metric
            </th>
            <th
              className="lg:py-3 md:py-2 lg:px-10 md:px-7 text-center text-[#6B7280] dark:text-dark-text-fill"
            >
              Grade
            </th>
            <th
              className="lg:py-3 md:py-2 text-left text-[#6B7280] dark:text-dark-text-fill"
            >
              Comment
            </th>
            <th
              className="lg:py-3 md:py-2"
            />
          </tr>
        </thead>
        <tbody
          className=" text-center "
        >
          <tr
            className="text-light-text dark:text-dark-text-fill bg-light-bg dark:bg-dark-bg "
          >
            <td
              className="lg:py-10 md:py-0 px-10 text-left  "
            >
              Quantity
            </td>
            <td
              className="py-3 "
            >
              1
            </td>
            <td
              className="lg:py-6 md:py-3 text-start lg:text-sm"
            >
              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam
            </td>
            <td
              className="py-3 px-8"
            >
              <button
                className="btn primary sm px-4 py-0 text-sm"
              >
                Reply
              </button>
            </td>
          </tr>
          <tr
            className="text-black dark:text-dark-text-fill bg-gray-100 dark:bg-dark-tertiary "
          >
            <td
              className="py-10 px-10 text-left"
            >
              Quality
            </td>
            <td
              className="py-3 "
            >
              1
            </td>
            <td
              className="py-3  text-start text-sm"
            >
              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam
            </td>
            <td
              className="py-3 "
            >
              <button
                className="btn primary sm px-4 py-0 text-sm"
              >
                Reply
              </button>
            </td>
          </tr>
          <tr
            className="text-black dark:text-dark-text-fill bg-light-bg dark:bg-dark-bg "
          >
            <td
              className="py-10 px-10 text-left"
            >
              Professional skills
            </td>
            <td
              className="py-3 "
            >
              1
            </td>
            <td
              className="py-3  text-start text-sm"
            >
              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam
            </td>
            <td
              className="py-3"
            >
              <button
                className="btn primary sm px-4 py-0 text-sm"
              >
                Reply
              </button>
            </td>
          </tr>
        </tbody>
      </table>
    </div>
    <div
      className="sm:flex sm:flex-col md:hidden justify-center items-center bg-light-bg dark:bg-dark-frame-bg dark:text-white  text-black"
    >
      <div
        className="flex flex-col justify-center items-start w-full py-4 px-4 ml-4 mt-4"
      >
        <h2
          className="font-semibold mt-12 "
        >
           
          Sprint 1
        </h2>
        <h3
          className="font-medium text-left text-[#6B7280] dark:text-white "
        >
          June 20, 2022 - June 24, 2022
        </h3>
      </div>
      <div
        className="flex flex-col justify-center items-center w-11/12 border shadow rounded-lg dark:border-dark-frame-bg bg-white  dark:bg-dark-bg pb-4 mt-4 pt-16"
      >
        <table
          className=" rounded-lg w-7/12 h-[70%] overflow-hidden  md:shadow mt-[-40px]"
        >
          <thead>
            <tr
              className="text-gray-300 dark:text-white  bg-gray-100 dark:bg-dark-tertiary "
            >
              <th
                className="py-2 px-3 text-center text-[#6B7280] dark:text-dark-text-fill "
              >
                Metric
              </th>
              <th
                className="py-2 px-10 text-center text-[#6B7280] dark:text-dark-text-fill"
              >
                Quantity
              </th>
            </tr>
          </thead>
          <tbody
            className=" text-center"
          >
            <tr
              className="text-light-text bg-light-bg dark:bg-dark-bg dark:text-dark-text-fill  "
            >
              <td
                className="py-3 px-10"
              >
                Grade
              </td>
              <td
                className=" "
              >
                1
              </td>
            </tr>
            <tr
              className="text-black bg-gray-100 dark:bg-dark-tertiary dark:text-dark-text-fill  "
            >
              <td
                className="py-10 px-10 text-left"
              >
                Comments
              </td>
              <td
                className="py-3  text-start text-sm"
              >
                Lorem ipsum dolor sit amet, consectetur adipiscing elit
              </td>
            </tr>
          </tbody>
        </table>
        <button
          className="px-4 py-1 ml-40 mt-4 rounded-md dark:text-dark-text-fill text-center bg-primary text-white  text-sm"
        >
          Reply
        </button>
      </div>
      <div
        className="flex flex-col justify-center items-center w-11/12 border shadow rounded-lg dark:border-dark-frame-bg bg-white dark:bg-dark-bg  pb-4 mt-4 pt-16"
      >
        <table
          className=" rounded-lg w-7/12 h-[70%] overflow-hidden  md:shadow mt-[-40px]"
        >
          <thead>
            <tr
              className="text-gray-300 dark:text-dark-text-fill bg-gray-100 dark:bg-dark-tertiary"
            >
              <th
                className="py-2 px-3 text-center text-[#6B7280] dark:text-dark-text-fill  "
              >
                Metric
              </th>
              <th
                className="py-2 px-10 text-center text-[#6B7280] dark:text-dark-text-fill"
              >
                Quality
              </th>
            </tr>
          </thead>
          <tbody
            className=" text-center"
          >
            <tr
              className="text-light-text bg-light-bg dark:bg-dark-bg dark:text-dark-text-fill "
            >
              <td
                className="py-3 px-10"
              >
                Grade
              </td>
              <td
                className=" "
              >
                1
              </td>
            </tr>
            <tr
              className="text-black bg-gray-100 dark:bg-dark-tertiary dark:text-dark-text-fill  "
            >
              <td
                className="py-10 px-10 text-left"
              >
                Comments
              </td>
              <td
                className="py-3  text-start text-sm"
              >
                Lorem ipsum dolor sit amet, consectetur adipiscing elit
              </td>
            </tr>
          </tbody>
        </table>
        <button
          className="px-4 py-1 ml-40 mt-4 rounded-md dark:text-dark-text-fill text-center bg-primary text-white  text-sm"
        >
          Reply
        </button>
      </div>
      <div
        className="flex flex-col justify-center items-center w-11/12 border shadow rounded-lg dark:border-dark-frame-bg bg-white dark:bg-dark-bg  pb-4 mt-4 pt-16"
      >
        <table
          className=" rounded-lg w-7/12 h-[70%] overflow-hidden  md:shadow mt-[-40px]"
        >
          <thead>
            <tr
              className="text-gray-300  bg-gray-100 dark:bg-dark-tertiary "
            >
              <th
                className="py-2 px-0 text-center text-[#6B7280] dark:text-dark-text-fill "
              >
                Metric
              </th>
              <th
                className="py-4 px-10 text-justify  text-[#6B7280] dark:text-dark-text-fill"
              >
                Professional skills
              </th>
            </tr>
          </thead>
          <tbody
            className=" text-center"
          >
            <tr
              className="text-light-text bg-light-bg dark-bg dark:bg-dark-bg dark:text-dark-text-fill "
            >
              <td
                className="py-3 px-10"
              >
                Grade
              </td>
              <td
                className=" "
              >
                1
              </td>
            </tr>
            <tr
              className="text-black bg-gray-100 dark:bg-dark-tertiary dark:text-dark-text-fill "
            >
              <td
                className="py-10 px-10 text-left"
              >
                Comments
              </td>
              <td
                className="py-3  text-start text-sm"
              >
                Lorem ipsum dolor sit amet, consectetur adipiscing elit
              </td>
            </tr>
          </tbody>
        </table>
        <button
          className="px-4 py-1 ml-40 mt-4 rounded-md dark:text-dark-text-fill text-center bg-primary text-white  text-sm"
        >
          Reply
        </button>
      </div>
    </div>
  </div>
  <div
    className="bg-neutral-100 dark:bg-dark-bg dark:text-white flex flex-col justify-start items-center  lg:ml-[-32px] md:ml-[-44px] sm:ml-[-24px] sm:pb-80 lg:mt-[-200px] sm:mt-[-200px] md:mt-[-550px] md:py-8 h-full"
  >
    <div
      className="w-7/12 flex flex-col md:py-10 sm:py-12 md:mt-0 lg:ml-0 md:ml-[-120px]  sm:mt-44"
    >
      <div
        className=" w-7/12 "
      >
        <h2
          className="font-md"
        >
          Comments
        </h2>
      </div>
      <div
        className=" mt-4"
      >
        <h2>
          Kenth Ngabo
        </h2>
        <p
          className="text-[#6B7280] dark:text-dark-text-fill"
        >
          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam
        </p>
      </div>
      <div
        className=" mt-4 ml-24"
      >
        <h2>
           John Doe
        </h2>
        <p
          className="text-[#6B7280] dark:text-dark-text-fill "
        >
          Lorem ipsum dolor sit amet, consectetur adipiscing elit
        </p>
        <div>
          <input
            className="bg-neutral-100 border-b border-gray-400   dark:bg-dark-bg outline-none"
            placeholder="Reply..."
            type="text"
          />
        </div>
      </div>
      <div>
        <div
          className="mt-4 "
        >
          <h2>
             John Doe
          </h2>
          <p
            className="text-[#6B7280] dark:text-dark-text-fill"
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit
          </p>
          <div
            className="ml-24 "
          >
            <input
              className="bg-neutral-100 border-b border-gray-400 dark:bg-dark-bg outline-none"
              placeholder="Reply..."
              type="text"
            />
          </div>
        </div>
      </div>
      <div
        className="mt-4 "
      >
        <input
          className="bg-neutral-100 border-b border-gray-400 dark:bg-dark-bg outline-none"
          placeholder="New comments..."
          type="text"
        />
      </div>
    </div>
  </div>
  <div
    className="bg-neutral-100 dark:bg-dark-frame-bg"
  >
    <button
      className="flex lg:ml-72  sm:ml-14 bg-primary px-4 md:py-2 sm:py-1 md:mt-0 sm:mt-[-200px]  rounded-md text-white font-semibold cursor-pointer"
      onClick={[Function]}
    >
      <svg
        className="mt-1 mr-1"
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        strokeWidth="0"
        style={
          Object {
            "color": undefined,
          }
        }
        viewBox="0 0 512 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M217.9 256L345 129c9.4-9.4 9.4-24.6 0-33.9-9.4-9.4-24.6-9.3-34 0L167 239c-9.1 9.1-9.3 23.7-.7 33.1L310.9 417c4.7 4.7 10.9 7 17 7s12.3-2.3 17-7c9.4-9.4 9.4-24.6 0-33.9L217.9 256z"
        />
      </svg>
      Back
    </button>
  </div>
</div>
`;
